suites/src/main/java/org/hiero/block/suites/BaseSuite.java

```diff
@@ -8,6 +8,7 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 import org.hiero.block.api.protoc.BlockAccessServiceGrpc;
+import org.hiero.block.api.protoc.BlockNodeServiceGrpc;
```

Comment:
Let's avoid the protoc usage.
With the new PBJ version you should be able to use the Service Interface generated.
```suggestion
import org.hiero.block.api..BlockNodeServiceInterface;
```

Reply:
Didn't we settle down to use protoc in the suites and simulator ? Not 100% sure anymore.

Reply:
We did settle on that (I think it is just easy to forget).
Suites and Simulator should act like "real" clients as much as possible, and almost all clients would be using SDKs derived using protoc.